.formatter.exs

```diff
@@ -1,4 +1,5 @@
 [
+  plugins: [Phoenix.LiveView.HTMLFormatter],
```

Comment:
https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.HTMLFormatter.html#module-setup

Reply:
It's already enabled here: https://github.com/firezone/firezone/blame/master/apps/fz_http/.formatter.exs#L16